api.graphql

```diff
@@ -0,0 +1,71 @@
+# Proposed interface
+# API gets 'msg' and 'medium'
+# 
+
+schema {
+  query: Query
+}
+
+# Message, Mode: (TARGET, ANNOUNCE)
+# - Announce:
+#   - Facebook, Twitter, Slack @announcements, Live Site
+# - Target:
+#   - Email, Twilio, Slacking PMs
+# message, group, and medium
+# for each medium, we have a mapping to a group
+# Email: - select user list via tags
+# Mobile: - same thing as email
+# Slack: - send to general with prepended/preformatted [TAG NAME]
+#        - or direct PMs
+# Tweet: No targetting - ( provide disclaimer )
+# Push notification: - can we target? api tokens
+type Query {
+  send_message(message: String!, plugins: PluginMaster!): PluginReturn!
+}
+
+#
+# enum StatusEnum {
+#   SUCCESS
+#   ERROR
+#   NULL
+# }
+
+type Status { # corresponding to APIReturn
+  error: Int!,
```

Comment:
What does this `Int!` mean?
Should be documented or replaced by a more descriptive type.